README.rst

```diff
@@ -24,7 +24,7 @@ Dependencies
 Required
 ````````
 * Python 2.7 (**note**: Python 3 is not yet supported)
-* `Django 1.8 to 1.10 (inclusive) <http://www.djangoproject.com/>`_
+* `Django 1.10 to 1.11 (inclusive) <http://www.djangoproject.com/>`_
```

Comment:
It looks like with only one change (in a few places) the tests will continue to pass back to Django 1.8. Does it seem worthwhile keeping that compatibility?  Django 1.8 is still in support until next April. (Though 1.9 is out of  support.) (https://www.djangoproject.com/download/#supported-versions)

Example change:

diff --git a/file_picker/sites.py b/file_picker/sites.py
index 9c27b0f..845de15 100644
--- a/file_picker/sites.py
+++ b/file_picker/sites.py
@@ -1,7 +1,7 @@
 import json
 
 from django.conf.urls import url
-from django.urls import reverse, NoReverseMatch
+from django.core.urlresolvers import reverse, NoReverseMatch
 from django.db import models
 from django.db.models.base import ModelBase, FieldDoesNotExist
 from django.http import HttpResponse